scripts/clusterfuzz/run.py

```diff
@@ -91,6 +104,34 @@ def get_js_file_contents(wasm_contents):
     # mechanism where the wasm file's name is provided in argv).
     wasm_contents = ','.join([str(c) for c in wasm_contents])
     js = f'var binary = new Uint8Array([{wasm_contents}]);\n\n' + js
+
+    # The default JS builds and runs the wasm. Append some random additional
+    # operations as well, as more compiles and executions can find things. To
+    # approximate a number in the range [0, MAX_EXTRA_JS_OPERATIONS) but with a
+    # mean of MEAN_EXTRA_JS_OPERATIONS, start with a number in [0, 1) and then
+    # raise it to the proper power, as multiplying by itself keeps the range
+    # unchanged, but lowers the mean. Specifically, the mean begins at 0.5, so
+    #
+    #   0.5^power = MEAN_EXTRA_JS_OPERATIONS / MAX_EXTRA_JS_OPERATIONS
+    #
+    # is what we want, and if we take log2 of each side, gives us
+    #
+    #   power =  log2(MEAN_EXTRA_JS_OPERATIONS / MAX_EXTRA_JS_OPERATIONS) / log2(0.5)
+    #         = -log2(MEAN_EXTRA_JS_OPERATIONS / MAX_EXTRA_JS_OPERATIONS)
```

Comment:
This doesn't actually produce a mean of `MEAN_EXTRA_JS_OPERATIONS`, does it?

This explanation (and in particular the origin of the 0.5) might be clearer if you explain that you want the _median_ to be `MEAN_EXTRA_JS_OPERATIONS` and the max to be `MAX_EXTRA_JS_OPERATIONS`.

Reply:
Oh yes, good point, I really meant the median here. E.g. the mean of `x` on `[0, 1)` is 0.5, and the mean of `x^2` is 0.333, while it is the median that goes to 0.25. Fixed.